docs/examples/academy/29. Working with Aggregated Cargo Movements Data.ipynb

```diff
@@ -147,41 +139,32 @@
       "text/plain": [
-       "<Figure size 576x432 with 1 Axes>"
+       "<Figure size 800x600 with 1 Axes>"
       ]
      },
-     "metadata": {
-      "needs_background": "light"
-     },
+     "metadata": {},
      "output_type": "display_data"
     }
    ],
    "source": [
-    "quantity_by_category = cms.groupby(by = ['loading_week','product_category']).sum().reset_index()\n",
+    "quantity_by_category = cms.groupby(by = ['loading_week','product_category']).sum(numeric_only=True).reset_index()\n",
```

Comment:
This is the only meaningful change in this file - it fixes an issue where the `.sum` method was not working after a Pandas upgrade.